examples/.explainer-config.toml

```diff
@@ -0,0 +1,9 @@
+# Any of these config options can or can not be provided

```

Comment:
this statement is incorrect. If the user choses to not define `apiKey` in `.toml` file it will not work. User either has to define it in `.env` file, in `.toml` file or via command line. It should be clarified which ones have default value defined already aka doesn't necessitate a definition for the tool to work and which ones can modified for user configuration.

Reply:
updated